aviary/docs/getting_started/onboarding_level1.ipynb

```diff
@@ -691,7 +687,7 @@
    "source": [
     "### Plain text inputs and outputs\n",
     "\n",
-    "If `debug_mode` is set to `True`, `input_list.txt` and `output_list.txt` are generated. We recommend this parameter be set to `True` for beginners. \n",
+    "If `Settings.VERBOSITY` is set to `2` or higher, `input_list.txt` and `output_list.txt` are generated. We recommend this parameter be set to `True` for beginners. \n",
```

Comment:
The last part of this sentence "recommend this parameter be set to 'True'" is no longer the case.  I don't know what verbosity we should recommend for beginners, but maybe not the highest.

Reply:
I believe brief is the default for everything, except tests which are Quiet

Reply:
Modified as:

If `Settings.VERBOSITY` is set to `2` (Verbosity.VERBOSE) or higher, `input_list.txt` and `output_list.txt` are generated. We recommend this parameter be set to `1` (Verbosity.BRIEF) for beginners.